Scanner.c is missing on npm, therefore preventing the package installation

The [scanner.c](https://github.com/DerekStride/tree-sitter-sql/blob/a65f58c364f4051e14c54be06dfdb91b546d6e7f/src/scanner.c) file is missing on the published [npm package](https://www.npmjs.com/package/@derekstride/tree-sitter-sql?activeTab=code). This is causing the `npm install` to fail during the `node-gyp rebuild` step.

## Steps to reproduce

```sh
$ mkdir tree-sitter-sql
$ cd tree-sitter-sql
$ npm pack @derekstride/tree-sitter-sql@0.1.1
$ tar --strip-components=1 -xvf derekstride-tree-sitter-sql-0.1.1.tgz package/
$ npm install
```

The compilation step will fail with the following error:

```sh
make: *** No rule to make target 'Release/obj.target/tree_sitter_sql_binding/src/scanner.o', needed by 'Release/obj.target/tree_sitter_sql_binding.node'.  Stop.
```

But if you download the `scanner.c` manually, it'll work:

```sh
$ mkdir -p src/
$ curl -Lo src/scanner.c 'https://derek.stride.host/tree-sitter-sql/src/scanner.c'
$ npm install
```

I'm not well versed on the *npm publish* behavior, but I think this is being caused by [having `/src/` defined on `.gitignore`](https://github.com/DerekStride/tree-sitter-sql/blob/a65f58c364f4051e14c54be06dfdb91b546d6e7f/.gitignore#L12).

--- a/.gitignore
+++ b/.gitignore
@@ -9,4 +9,6 @@ build
 package-lock.json
 /target/
 /.build/
-/src/
+/src/tree_sitter/
+/src/parser.c
+/src/*.json

--- a/README.md
+++ b/README.md
@@ -2,6 +2,8 @@
 
 [![Build/test](https://github.com/derekstride/tree-sitter-sql/actions/workflows/ci.yml/badge.svg)](https://github.com/derekstride/tree-sitter-sql/actions/workflows/ci.yml)
 [![GitHub Pages](https://github.com/DerekStride/tree-sitter-sql/actions/workflows/gh-pages.yml/badge.svg)](https://github.com/DerekStride/tree-sitter-sql/actions/workflows/gh-pages.yml)
+[![npm package version](https://img.shields.io/npm/v/%40derekstride/tree-sitter-sql?logo=npm&color=brightgreen)](https://www.npmjs.com/package/@derekstride/tree-sitter-sql)
+
 
 A general/permissive SQL grammar for [tree-sitter](https://github.com/tree-sitter/tree-sitter).
 

